pkg/nsm/apis/nsmconnect/nsmconnect.proto

```diff
@@ -27,7 +27,7 @@ message ConnectionReply {
     bool accepted = 1;
     string admission_error = 2;
     ConnectionParameters connection_parameters = 3;
-    common.Interface interface = 4;
+    common.LocalMechanism interface = 4;
```

Comment:
ditto

Reply:
Yes, was going to do that subsequently (I'm going in small steps). Happy to do it in this patch if you prefer.